platform.md

```diff
@@ -605,14 +605,15 @@ When extending the build image:
 | `1-10`, `13-19` | Generic lifecycle errors            |
 | `100-109`       | Extension-specific lifecycle errors |
 
-- For each extension in `<group>` in order, if a Dockerfile exists in `<generated>/<kind>/<buildpack-id>`, the lifecycle:
+- For each extension in `<group>` in order, if a Dockerfile exists in `<generated>/<buildpack-id>/<kind>.Dockerfile`, the lifecycle:
   - SHALL apply the Dockerfile to the environment according to the process outlined in the [Image Extension Specification](image-extension.md).
+  - SHALL set the Kaniko build context to the folder according to the process outlined in the [Image Extension Specification](image-extension.md).
```

Comment:
```suggestion
  - SHALL set the build context to the folder according to the process outlined in the [Image Extension Specification](image-extension.md).
```

Reply:
In addition I think we can remove these lines from the detector:

```
SHALL copy any generated run.Dockerfiles to <generated>/run/<image extension ID>/Dockerfile.
SHALL copy any generated build.Dockerfiles to <generated>/build/<image extension ID>/Dockerfile.
SHALL copy any generated <extend-config> files to <generated>/build/<image extension ID>/<extend-config>.
```

(as buildpacks are already writing Dockerfiles to the extender's expected location)

Could we also add something like:
> If any `build.Dockerfiles` exist, SHALL record `build-image.extend = true` in `<analyzed>`

somewhere after `When image extensions are present in the order (optional and experimental), the lifecycle:` in the detector outputs